src/codemirrorCommands.ts

```diff
@@ -8,16 +8,26 @@ import { CommandRegistry } from '@lumino/commands';
  */
 const IS_MAC = !!navigator.platform.match(/Mac/i);
 
+export interface Keybinding {
+  command: string
+  keys: string
+  context: string
+  mapfn: string
+  enabled: boolean
+}
+
 export class VimCellManager {
   constructor(
     commands: CommandRegistry,
     cm: CodeMirrorEditor,
-    enabled: boolean
+    enabled: boolean,
+    userKeybindings: any
```

Comment:
```suggestion
    userKeybindings: IKeybinding[]
```

Reply:
Since this is a breaking change to exported member, we could do one of two things:
- make the argument optional so that it is non-breaking and release a minor version, or
- take the opportunity and rewrite the constructor to use the `options: IOptions` pattern which packs all arguments into an interface which is nicer to work with and does not require breaking changes when things get removed.

Reply:
Good point. It sounds like the best thing to do is define an `IOptions` interface, so I'll do that.